docs/v0.7/en/release-notes/all-releases.md

```diff
@@ -0,0 +1,5 @@
+# All releases
```

Comment:
I'd like to use differnt title from the sidebar text. Currently, they are coupled.

@alili @Aganivi do you have time to implement this feature? I guess with a sidebarLabel YAML frontmatter it can be distinguished.

Reply:
For example, the sidebar label of this page should be "All releases", but for page title, I'd prefer "GreptimeDB Release Notes".

Reply:
I think it is a little strange that the navigation title is different with the page title.
Cloud you please explain why do you want this feature? Is there any advantage? Maybe we can use “Release Notes” in both the navigation title and  the page title.

Reply:
"All Releases" may be good, but "v0.7.0" can be better as "GreptimeDB v0.7.0 Release Notes" as its title.

Take https://docs.pingcap.com/tidb/stable/release-7.5.1 as an example.